packages/styles/settings-tools/_s.notification.scss

```diff
@@ -0,0 +1,37 @@
+$theme-background-colors: (
+  'information': $color-secondary-blue-100,
+  'success': $color-primary-01-100,
+  'warning': $color-secondary-orange-100,
+  'danger': $color-secondary-red-100,
+);
+$theme-border-icon-colors-border: (
+  'information': (
+    'color': $color-secondary-blue-500,
+    'icon': 'notification-information-48',
+  ),
+  'success': (
+    'color': $color-primary-01-500,
+    'icon': 'notification-success-24',
+  ),
+  'warning': (
+    'color': $color-secondary-orange-500,
+    'icon': 'notification-warning-24',
+  ),
+  'danger': (
+    'color': $color-secondary-red-500,
+    'icon': 'notification-danger-48',
+  ),
+);
```

Comment:
The best way to add colors is used token files in json for colors. You have to create one.